build.gradle.kts

```diff
@@ -637,6 +637,127 @@ tasks.register("formatChanges") {
   }
 }
 
+tasks.register("validateChanges") {
```

Comment:
Could we make this a precondition of another task which runs linting? Maybe whitespacePreCommit?

That way it runs with precommits

Reply:
sounds good. I will do this with another PR for the workflow.